tests/test_element_list.py

```diff
@@ -102,3 +103,15 @@ def test_not_found_exception_with_query_and_method(self):
         expected_message = 'no elements could be found with id "menu"'
         e = cm.exception
         self.assertEqual(expected_message, e.args[0])
+
+    def test_method_get(self):
+        """
+        should provide a \"get\" method
+        which provides access to HTML attributes' element list of the visited webpage
+        """
+        wd_ins = splinter.Browser()
```

Comment:
You don't need a live browser instance to test getting attributes. You can just use a WebDriverElement instance.